src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -78,6 +92,44 @@ protected override void BeginProcessing()
         {
             string resolvedpath = string.Empty;
 
+            try
+            {
+                if (Json != null)
+                {
+                    _json = Json;
+                }
+                else if (JsonFile != null)
+                {
+                    try
+                    {
+                        resolvedpath = Context.SessionState.Path.GetUnresolvedProviderPathFromPSPath(JsonFile);
+                        // _json = JsonFile.FromFileAsync(resolvedpath).Result;
+                        _json = System.IO.File.ReadAllText(resolvedpath);
+                    }
+                    catch (AggregateException ae)
+                    {
+                        ae.Handle(UnwrapException);
+                    }
+                }
+            }
+            catch (Exception e) when (
+                // Handle exceptions related to file access to provide more specific error message
+                // https://docs.microsoft.com/en-us/dotnet/standard/io/handling-io-errors
+                e is IOException ||
+                e is UnauthorizedAccessException ||
+                e is NotSupportedException ||
+                e is SecurityException
+            )
```

Comment:
```suggestion
            catch (Exception e) when (
                // Handle exceptions related to file access to provide more specific error message
                // https://docs.microsoft.com/en-us/dotnet/standard/io/handling-io-errors
                e is IOException ||
                e is UnauthorizedAccessException ||
                e is NotSupportedException ||
                e is SecurityException)
```

I think this is what's tripping up the linters; C# tends to expect closing parentheses on the same line as the last line of code (unlike braces; not really sure why they're generally different but it seems to be the case).